classes/search/activity.php

```diff
@@ -0,0 +1,137 @@
+<?php
+// This file is part of Moodle - http://moodle.org/
+//
+// Moodle is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// Moodle is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with Moodle.  If not, see <http://www.gnu.org/licenses/>.
+
+namespace mod_cms\search;
+
+defined('MOODLE_INTERNAL') || die();
+
+use mod_cms\local\model\cms;
+use mod_cms\local\renderer;
+
+require_once($CFG->dirroot . '/mod/cms/lib.php');
```

Comment:
why do we need this lib?

Reply:
It looks the required file is no longer needed.
I removed the required file, and tested ok.